src/app/components/form/form.component.html

```diff
@@ -0,0 +1,30 @@
+<form class="form" [formGroup]="postForm" (ngSubmit)="onSubmit()">
```

Comment:
good job, but please, use better naming, for example, article-form.component

Reply:
Good. Thanks. Fixed